server.js

```diff
@@ -12,7 +12,7 @@ const PORT = process.env.PORT || 8080;
 
 // Requiring our models for syncing
 const db = require("./models");
-
+//const seed = require("./db/seed.js");
```

Comment:
We don't need this for the app. If you want to test dummy data, you can try drafting up a seed.sql